package.json

```diff
@@ -29,7 +29,7 @@
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "build-css": "parcel build styles/themes/default.scss --out-dir dist/themes/",
-    "build": "npm run test && npm run clean && npm run build-css && rollup -c --bundleConfigAsCjs",
+    "build": "npm --max_old_space_size=4096 run test && npm run clean && npm run build-css && rollup -c --bundleConfigAsCjs",
```

Comment:
not even sure if this works in all situations (did not work on Linux), we should add a note to this for the future